doc/src/modules/physics/mechanics/symsystem.rst

```diff
@@ -0,0 +1,196 @@
+=====================================
+Symbolic Systems in Physics/Mechanics
+=====================================
+
+The `SymbolicSystem` class in physics/mechanics is a location for the pertinent
+information of a multibody dynamic system. In its most basic form it contains
+the equations of motion for the dynamic system, however, it can also contain
+information regarding the loads that the system is subject to, the bodies that
+the system is comprised of and any additional equations the user feels is
+important for the system. The goal of this class is to provide a unified output
+format for the equations of motion that numerical analysis code can be designed
+around.
+
+SymbolicSystem Example Usage
+============================
+
+This code will go over the manual input of the equations of motion for the
+simple pendulum that uses the Cartesian location of the mass as the generalized
+coordinates into `SymbolicSystem`.
+
+The equations of motion are formed in the physics/mechanics/examples_. In that
+spot the variables q1 and q2 are used in place of x and y and the reference
+frame is rotated 90 degrees.
+
+.. _examples: ../examples/lin_pend_nonmin_example.html
+
+::
+
+    >>> from sympy import atan, symbols, Matrix
+    >>> from sympy.physics.mechanics import (dynamicsymbols, ReferenceFrame,
+    ...                                      Particle, Point)
+    >>> import sympy.physics.mechanics.system as system
+
+The first step will be to initialize all of the dynamic and constant symbols. ::
+
+    >>> x, y, u, v, lam = dynamicsymbols('x y u v lambda')
+    >>> m, l, g = symbols('m l g')
+
+Next step is to define the equations of motion in multiple forms:
+
+    [1] Explicit form where the kinematics and dynamics are combined
+        x' = F_1(x, t, r, p)
+
+    [2] Implicit form where the kinematics and dynamics are combined
+        M_2(x, p) x' = F_2(x, t, r, p)
+
+    [3] Implicit form where the kinematics and dynamics are separate
+        M_3(q, p) u' = F_3(q, u, t, r, p)
+        q' = G(q, u, t, r, p)
+
```

Comment:
I think it would be nice to introduce the variable naming convention we have in the class near these definitions so that it is easily found and understood. Maybe in the list below you can note what attribute name is associated with what variable in these 3 equations.